### ISSUE TITLE
Deserialization problem when not using any config file

### ISSUE THREAD

**Issue by gustavkarlsson** [dropwizard-bundles/dropwizard-configurable-assets-bundle#13]
When running my DW application without a config file argument, I get:

```
default configuration has an error:
  * Failed to parse configuration at: assets.overrides; Can not deserialize instance of java.util.LinkedHashMap out of START_ARRAY token
 at [Source: N/A; line: -1, column: -1] (through reference chain: my.project.ProjectConfiguration["assets"]->io.dropwizard.bundles.assets.AssetsConfiguration["overrides"])
```

If I use a config file (even one that is lacking an assets section!), everything works as expected. This prevents me from having a default configuration since the overrides are unmodifiable and can't be set from within the Configuration class.

**Comment by nbauernfeind** [comment]
Ok, I see the issue. I wonder why dropwizard does default configuration this way. It appears to create the default object, dump it out to JSON, and then deserialize the JSON into a config object. Since `getOverrides` returns an `Iterable<Map.Entry<String, String>>` instead of the Map then it finds a discrepancy looking to parse a map but finds the array/iterable instead.

**Comment by nbauernfeind** [comment]
This has been released in version `0.8.1-1`. Thanks for pointing it out!
